### DISCUSSION TITLE
Saving Players isn't working.

### DISCUSSION THREAD

**Discussion by YXZory** [luau/UniversalSynSaveInstance#54] [Q&A] [answered]
So, I've been trying to save the players so I can get the GUI. But I have read the options and it confuses me. Could anyone give me a simple answer?

**Comment by phoriah** [comment, marked as answer]
> So, I've been trying to save the players so I can get the GUI. But I have read the options and it confuses me. Could anyone give me a simple answer?

Add `SavePlayers=true` to Options table.

**Reply by YXZory** [reply]
Thank you!
